potodo/_github.py

```diff
@@ -33,14 +33,23 @@ def get_repo_name() -> str:
 
 
 def get_reservation_list() -> Mapping[str, str]:
-    """Will get the repository name then request all the issues and put them in a dict
     """
-    # Gets the issues into a dict
-    issues = requests.get(
-        "https://api.github.com/repos/" + get_repo_name() + "/issues"
-    ).json()
-    # Creates the issue dict with the name and the login
-    reservations: dict = {
-        issue["title"].split()[-1].lower(): issue["user"]["login"] for issue in issues
-    }
+    Will get the repository name then request all the issues and put them in a dict
+    """
+
+    issues: list = []
+    next = "https://api.github.com/repos/" + get_repo_name() + "/issues?state=open"
```

Comment:
You're overwriting the `next` builtin here, maybe next was not a good variable name (sorry I bet It's my fault).